### ISSUE TITLE
Action fails if the branch already exists

### ISSUE THREAD

**Issue by timbru31** [ybiquitous/npm-audit-fix-action#884]
The action sadly fails if the branch name already exists (`npm-audit-fix-action/fix`) - maybe because after merging a fix PR the branch was not deleted:

```
Create or update a pull request
  /usr/bin/git config user.name timbru31
  /usr/bin/git config user.email timbru31@users.noreply.github.com
  /usr/bin/git add package-lock.json
  /usr/bin/git commit --message build(deps): npm audit fix
  
  Summary:
  - Updated packages: 2
  - Added packages: 0
  - Removed packages: 0
  
  No fixed vulnerabilities.
  
  [master b4151a8] build(deps): npm audit fix
   1 file changed, 10 insertions(+), 8 deletions(-)
  /usr/bin/git checkout -B npm-audit-fix-action/fix
  Switched to a new branch 'npm-audit-fix-action/fix'
  /usr/bin/git push ***github.com/timbru31/sad-creeper.git HEAD:npm-audit-fix-action/fix
  To https://github.com/timbru31/sad-creeper.git
   ! [rejected]        HEAD -> npm-audit-fix-action/fix (fetch first)
  error: failed to push some refs to 'https://github.com/timbru31/sad-creeper.git'
  hint: Updates were rejected because the remote contains work that you do not
  hint: have locally. This is usually caused by another repository pushing to
  hint: the same ref. If you want to integrate the remote changes, use
  hint: 'git pull' before pushing again.
  hint: See the 'Note about fast-forwards' in 'git push --help' for details.
Error: The process '/usr/bin/git' failed with exit code 1
```

**Comment by ybiquitous** [comment]
@timbru31 Thanks for the report! Do you want this action to delete the existing branch automatically?

If deleting, we have to delete the branch before push here:

https://github.com/ybiquitous/npm-audit-fix-action/blob/557f1be72c6c8504c24a36ba37b43b818730d597/lib/createOrUpdatePullRequest.js#L53

**Comment by timbru31** [comment]
I would be fine with that!

**Comment by ybiquitous** [comment]
Okay, I'll fix this. 👍🏼

**Comment by ybiquitous** [comment]
https://github.com/ybiquitous/npm-audit-fix-action/releases/tag/v6.1.2 is out. Can you take a look? Thanks. 😄
